src/test/java/io/xstefank/wildfly/bot/PRRuleTitleBodyCheckTest.java

```diff
@@ -122,4 +124,77 @@ void testFailedTitleBodyCheck() throws IOException {
                     Util.verifyFormatFailure(repo, gitHubJson, "title");
                 });
     }
+
+    @Test
+    void testTitleBodyCheckTitleRegex() throws IOException {
+        GitHubJson gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON)
+                .title("[WFLY-00000] Metrics PR")
+                .action(Action.OPENED)
+                .build();
+        wildflyConfigFile = """
+                wildfly:
+                  rules:
+                    - titleBody: "metrics|micrometer"
+                      notify: [7125767235]
+                  format:
+                    commit:
+                      enabled: false
+                """;
+        given().github(mocks -> Util.mockRepo(mocks, wildflyConfigFile, gitHubJson, null))
+                .when().payloadFromString(gitHubJson.jsonString())
+                .event(GHEvent.PULL_REQUEST)
+                .then().github(mocks -> {
+                    GHPullRequest mockedPR = mocks.pullRequest(gitHubJson.id());
+                    Mockito.verify(mockedPR).comment("/cc @7125767235");
+                });
+    }
+
+
+    @Test
+    void testTitleBodyCheckBodyRegex() throws IOException {
+        GitHubJson gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON)
+                .description("[WFLY-00000] Micrometer commit")
+                .action(Action.OPENED)
+                .build();
+        wildflyConfigFile = """
+                wildfly:
+                  rules:
+                    - titleBody: "metrics|micrometer"
+                      notify: [7125767235]
+                  format:
+                    commit:
+                      enabled: false
+                """;
+        given().github(mocks -> Util.mockRepo(mocks, wildflyConfigFile, gitHubJson, null))
+                .when().payloadFromString(gitHubJson.jsonString())
+                .event(GHEvent.PULL_REQUEST)
+                .then().github(mocks -> {
+                    GHPullRequest mockedPR = mocks.pullRequest(gitHubJson.id());
+                    Mockito.verify(mockedPR).comment("/cc @7125767235");
+                });
+    }
+
+    @Test
+    void testTitleBodyCheckBodyRegexNoHit() throws IOException {
+        GitHubJson gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON)
```

Comment:
Here the `GitHubJson gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON).build()` should be fine as default JSON description or title doesn't contain words "not" or "valid". And the action is "opened"